src/github.com/getlantern/elevate/command.go

```diff
@@ -0,0 +1,37 @@
+// package elevate provides support for executing commands with elevated
+// privileges.
+package elevate
+
+import (
+	"os/exec"
+)
+
+// Command is like exec.Command, except that it runs the given command with
+// elevated privileges.
+func Command(name string, args ...string) *exec.Cmd {
+	cmd, err := buildCommand("", "", name, args...)
+	if err != nil {
+		panic(err)
```

Comment:
The only time there would be an error is if we couldn't extract cocoasudo.  That's not a recoverable situation, so I figured I'd keep the interface cleaner.